### DISCUSSION TITLE
is it possible to code obfuscated all js files ?

### DISCUSSION THREAD

**Discussion by AhmadAlYaaqba** [vercel/next.js#17527] [Help]
i am looking for a way to code obfuscated all js files that generated for a better security 

i was looking in doc but nothing mentioned if this is possible 

do anyone have any idea to do this ?

**Comment by lfades** [comment]
That won't improve the security of your application, JS must be public in order for the browser to get access to them. If you want to improve your security the don't expose to the browser anything that's secret.

**Reply by AhmadAlYaaqba** [reply]
@lfades 
that would help at least to not make your code readable as much as possible, does not that help you?
